### PR TITLE
Add refresh views method to context

### PR DESCRIPTION
**Pull request by lask**

### CODE REVIEW

File: vscode/src/organization.ts [hunk 1]
```diff
@@ -23,11 +23,15 @@ async function updateStatus(ctx: Context) {
 }
 
 async function executeCommand(ctx: Context) {
-  await ensureAuth(ctx);
-  const org = await selectOrganization(ctx);
-  await setOrganization(ctx, org);
-  await updateStatus(ctx);
-  ctx.refreshDeviceView();
+  try {
```
Comment (floitsch) [inline vscode/src/organization.ts:26]: Seems unrelated.

**Review by floitsch — APPROVED** [review]
LGTM.

File: vscode/src/organization.ts (hunk 1 above)
Comment (lask) [reply vscode/src/organization.ts]: Must have forgotten to use the correct base.
